src/redux/services/fileEntry.ts

```diff
@@ -188,15 +190,24 @@ export function readFiles(
     );
   } else {
     files.forEach(file => {
+      let text;
       const filePath = path.join(folder, file);
       const fileEntryPath = filePath.substring(rootFolder.length);
-      const fileEntry = createFileEntry({fileEntryPath, fileMap, helmChartId: helmChart?.id});
+
+      const isDir = getFileStats(filePath)?.isDirectory();
+
+      if (!isDir) {
+        text = fs.readFileSync(path.join(filePath), 'utf8');
```

Comment:
I think because we now read and save the text content of the file on the fileEntry, then in the `extractK8sResourcesFromFile` function from `services/fileEntry.ts` we should use this text directly instead of reading those files again. Because we don't want to read all files that contain resources two times instead of only once.

Reply:
good point! reused the text